### ISSUE TITLE
Unknown CMake command "add_compile_definitions".

### ISSUE THREAD

**Issue by Dhciolfi** [facebookresearch/habitat-sim#137]
## Steps to reproduce

  1. Run sudo python3 setup.py install

## Observed Results

```
Building with ptex support
CMake Error at CMakeLists.txt:52 (add_compile_definitions):
Unknown CMake command "add_compile_definitions".
```

## Expected Results

  * It should setup properly

It seems that this area of code was modified in the last commit.

**Comment by mosra** [comment]
Whoops, missed this one when reviewing #133. Fix for this is in #139.

**Comment by mosra** [comment]
Fix merged to master. To prevent issues like this happening in the future, the CI now explicitly checks for compatibility back to CMake 3.10.
